README.md

```diff
@@ -131,6 +131,34 @@ RAILS_ENV=production bundle exec rake m2c_exist_all_storage_roots[profile]
 ```
 this will generate a log at, for example, ```log/profile_check_existence_for_all_storage_roots2017-12-11T14:25:31-flat.txt```
 
+### Run Catalog to Moab existence check for a single root and for all storage roots
```

Comment:
"for a single root or for all storage roots"  (or instead of and??)

Given that Julian and Vivian both expressed their confusion about what C2M is, perhaps either the heading or the first sentence after it should explain something like 
```
"given a catalog entry for an online moab, ensure that the online moab exists and that the catalog version matches the online moab version."
```